[2019-08-28 15:00:05,154] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:00:05,156] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:00:07,130] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:00:07,131] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:00:14,412] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:00:14,412] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:00:14,413] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:00:14,414] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:00:14,414] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:00:14,414] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:00:20,463] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:00:20,463] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:00:20,464] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:00:20,464] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:00:20,464] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:00:20,464] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:00:28,450] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:00:28,450] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:00:28,451] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:00:28,451] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:00:28,452] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:00:28,452] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:00:34,492] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:00:34,492] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:00:34,493] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:00:34,493] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:00:34,493] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:00:34,493] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:00:56,633] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:00:56,633] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:00:56,634] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:00:56,635] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:00:56,636] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:00:56,636] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:01:05,699] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:01:05,699] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:01:05,699] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:01:05,700] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:01:05,700] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:01:05,700] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:01:21,012] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:01:21,032] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:01:21,033] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:01:27,057] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:01:27,057] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:01:27,057] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:01:37,707] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:01:37,708] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:01:45,174] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:01:45,174] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:01:45,175] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:01:59,060] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:01:59,081] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:02:06,556] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:02:06,556] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:02:06,557] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:02:12,684] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:02:12,685] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:02:12,685] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:02:17,180] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:02:17,181] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:02:18,804] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:02:18,806] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:02:18,806] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:02:25,342] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:02:25,342] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:02:25,342] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:02:31,903] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:02:31,904] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:02:31,904] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:02:37,925] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:02:37,925] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:02:37,926] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:02:49,193] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:02:49,194] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:02:56,135] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:02:56,135] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:02:56,136] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:03:03,155] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:03:03,155] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:03:03,156] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:03:09,935] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:03:09,937] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:03:10,185] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:03:10,186] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:03:10,187] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:03:17,229] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:03:17,230] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:03:17,231] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:03:23,345] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:03:23,345] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:03:23,346] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:03:29,372] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:03:29,372] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:03:29,373] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:03:35,392] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:03:35,392] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:03:35,392] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:03:42,190] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:03:42,191] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:03:57,396] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:03:57,397] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:03:57,398] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:04:03,486] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:04:03,486] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:04:03,487] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:04:03,488] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:04:03,488] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:04:03,488] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:04:24,826] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:04:24,826] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:04:24,827] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:04:35,494] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:04:35,495] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:04:42,954] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:04:42,954] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:04:42,955] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:04:55,837] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-08-28 15:04:55,839] INFO [KafkaServer id=0] shutting down (kafka.server.KafkaServer)
[2019-08-28 15:04:55,844] INFO [KafkaServer id=0] Starting controlled shutdown (kafka.server.KafkaServer)
[2019-08-28 15:04:56,060] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-08-28 15:04:56,236] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-08-28 15:04:56,412] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-08-28 15:04:56,839] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:04:56,839] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:05:01,141] WARN [KafkaServer id=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:05:01,143] WARN [KafkaServer id=0] Error during controlled shutdown, possibly because leader movement took longer than the configured controller.socket.timeout.ms and/or request.timeout.ms: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed. (kafka.server.KafkaServer)
[2019-08-28 15:05:04,147] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:05:04,147] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:05:04,148] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:05:04,148] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:05:04,148] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:05:04,148] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:05:06,148] WARN [KafkaServer id=0] Retrying controlled shutdown after the previous attempt failed... (kafka.server.KafkaServer)
[2019-08-28 15:05:36,155] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:05:36,156] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:05:36,155] WARN [KafkaServer id=0] Error during controlled shutdown, possibly because leader movement took longer than the configured controller.socket.timeout.ms and/or request.timeout.ms: Failed to connect within 30000 ms (kafka.server.KafkaServer)
[2019-08-28 15:05:36,156] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:05:36,157] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:103)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:05:41,162] WARN [KafkaServer id=0] Retrying controlled shutdown after the previous attempt failed... (kafka.server.KafkaServer)
[2019-08-28 15:05:43,357] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:05:43,358] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:05:43,358] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:05:43,358] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:05:43,358] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:05:43,359] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:05:50,420] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:05:50,479] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:05:50,479] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:05:50,420] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:05:50,745] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:05:50,746] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:05:59,662] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:05:59,662] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:05:59,663] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:05:59,663] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:05:59,664] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:05:59,664] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:06:04,833] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2019-08-28 15:06:06,688] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:06:06,689] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:06:06,690] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:06:06,690] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:06:06,689] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:06:06,692] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:06:11,176] WARN [KafkaServer id=0] Error during controlled shutdown, possibly because leader movement took longer than the configured controller.socket.timeout.ms and/or request.timeout.ms: Failed to connect within 30000 ms (kafka.server.KafkaServer)
[2019-08-28 15:06:12,758] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Connection to node 1 (/192.168.1.13:9093) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:06:12,758] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Connection to node 2 (/192.168.1.13:9094) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:06:12,882] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:06:12,882] WARN [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={testKafka-1=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), testKafka1-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), test-kafka-broker-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[54]), topic127-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[47]), topic130-0=(fetchOffset=354, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[43]), topic125-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1007848171, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:06:12,882] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:06:12,886] WARN [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=0, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={topic130-2=(fetchOffset=231, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), topic127-0=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-3=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic128-0=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[36]), topic125-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka1-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), testKafka-2=(fetchOffset=1, logStartOffset=1, maxBytes=1048576, currentLeaderEpoch=Optional[67]), topic127-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[48]), Multibrokerapplication-0=(fetchOffset=5, logStartOffset=5, maxBytes=1048576, currentLeaderEpoch=Optional[72]), Multibrokerapplication-1=(fetchOffset=3, logStartOffset=3, maxBytes=1048576, currentLeaderEpoch=Optional[74]), topic130-1=(fetchOffset=342, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[42]), testKafka-0=(fetchOffset=4, logStartOffset=4, maxBytes=1048576, currentLeaderEpoch=Optional[68]), topic132-0=(fetchOffset=15, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), testKafka1-2=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), topic131-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[28]), topic125-0=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[50]), test-kafka-broker-1=(fetchOffset=0, logStartOffset=0, maxBytes=1048576, currentLeaderEpoch=Optional[58])}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=2037434846, epoch=INITIAL), rackId=) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.1.13:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:71)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:102)
	at kafka.server.ReplicaFetcherThread.fetchFromLeader(ReplicaFetcherThread.scala:197)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:286)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3(AbstractFetcherThread.scala:133)
	at kafka.server.AbstractFetcherThread.$anonfun$maybeFetch$3$adapted(AbstractFetcherThread.scala:132)
	at scala.Option.foreach(Option.scala:274)
	at kafka.server.AbstractFetcherThread.maybeFetch(AbstractFetcherThread.scala:132)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:114)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2019-08-28 15:06:16,181] WARN [KafkaServer id=0] Retrying controlled shutdown after the previous attempt failed... (kafka.server.KafkaServer)
[2019-08-28 15:06:16,188] WARN [KafkaServer id=0] Proceeding to do an unclean shutdown as all the controlled shutdown attempts failed (kafka.server.KafkaServer)
[2019-08-28 15:06:16,191] INFO [/config/changes-event-process-thread]: Shutting down (kafka.common.ZkNodeChangeNotificationListener$ChangeEventProcessThread)
[2019-08-28 15:06:16,195] INFO [/config/changes-event-process-thread]: Stopped (kafka.common.ZkNodeChangeNotificationListener$ChangeEventProcessThread)
[2019-08-28 15:06:16,200] INFO [/config/changes-event-process-thread]: Shutdown completed (kafka.common.ZkNodeChangeNotificationListener$ChangeEventProcessThread)
[2019-08-28 15:06:16,204] INFO [SocketServer brokerId=0] Stopping socket server request processors (kafka.network.SocketServer)
[2019-08-28 15:06:16,225] INFO [SocketServer brokerId=0] Stopped socket server request processors (kafka.network.SocketServer)
[2019-08-28 15:06:16,227] INFO [data-plane Kafka Request Handler on Broker 0], shutting down (kafka.server.KafkaRequestHandlerPool)
[2019-08-28 15:06:16,231] INFO [data-plane Kafka Request Handler on Broker 0], shut down completely (kafka.server.KafkaRequestHandlerPool)
[2019-08-28 15:06:16,236] INFO [KafkaApi-0] Shutdown complete. (kafka.server.KafkaApis)
[2019-08-28 15:06:16,238] INFO [ExpirationReaper-0-topic]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:16,329] INFO [ExpirationReaper-0-topic]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:16,329] INFO [ExpirationReaper-0-topic]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:16,333] INFO [TransactionCoordinator id=0] Shutting down. (kafka.coordinator.transaction.TransactionCoordinator)
[2019-08-28 15:06:16,334] INFO [ProducerId Manager 0]: Shutdown complete: last producerId assigned 51000 (kafka.coordinator.transaction.ProducerIdManager)
[2019-08-28 15:06:16,335] INFO [Transaction State Manager 0]: Shutdown complete (kafka.coordinator.transaction.TransactionStateManager)
[2019-08-28 15:06:16,335] INFO [Transaction Marker Channel Manager 0]: Shutting down (kafka.coordinator.transaction.TransactionMarkerChannelManager)
[2019-08-28 15:06:16,335] INFO [Transaction Marker Channel Manager 0]: Stopped (kafka.coordinator.transaction.TransactionMarkerChannelManager)
[2019-08-28 15:06:16,336] INFO [Transaction Marker Channel Manager 0]: Shutdown completed (kafka.coordinator.transaction.TransactionMarkerChannelManager)
[2019-08-28 15:06:16,337] INFO [TransactionCoordinator id=0] Shutdown complete. (kafka.coordinator.transaction.TransactionCoordinator)
[2019-08-28 15:06:16,338] INFO [GroupCoordinator 0]: Shutting down. (kafka.coordinator.group.GroupCoordinator)
[2019-08-28 15:06:16,340] INFO [ExpirationReaper-0-Heartbeat]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:16,529] INFO [ExpirationReaper-0-Heartbeat]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:16,529] INFO [ExpirationReaper-0-Heartbeat]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:16,530] INFO [ExpirationReaper-0-Rebalance]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:16,730] INFO [ExpirationReaper-0-Rebalance]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:16,730] INFO [ExpirationReaper-0-Rebalance]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:16,817] INFO [GroupCoordinator 0]: Shutdown complete. (kafka.coordinator.group.GroupCoordinator)
[2019-08-28 15:06:16,820] INFO [ReplicaManager broker=0] Shutting down (kafka.server.ReplicaManager)
[2019-08-28 15:06:16,916] INFO [LogDirFailureHandler]: Shutting down (kafka.server.ReplicaManager$LogDirFailureHandler)
[2019-08-28 15:06:16,917] INFO [LogDirFailureHandler]: Shutdown completed (kafka.server.ReplicaManager$LogDirFailureHandler)
[2019-08-28 15:06:16,917] INFO [LogDirFailureHandler]: Stopped (kafka.server.ReplicaManager$LogDirFailureHandler)
[2019-08-28 15:06:17,061] INFO [ReplicaFetcherManager on broker 0] shutting down (kafka.server.ReplicaFetcherManager)
[2019-08-28 15:06:17,063] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Shutting down (kafka.server.ReplicaFetcherThread)
[2019-08-28 15:06:17,064] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Shutting down (kafka.server.ReplicaFetcherThread)
[2019-08-28 15:06:17,064] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=2037434846, epoch=INITIAL) to node 1: org.apache.kafka.common.errors.DisconnectException: NetworkClient is no longer active, state is CLOSING. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:06:17,065] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Stopped (kafka.server.ReplicaFetcherThread)
[2019-08-28 15:06:17,065] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=1007848171, epoch=INITIAL) to node 2: org.apache.kafka.common.errors.DisconnectException: NetworkClient is no longer active, state is CLOSING. (org.apache.kafka.clients.FetchSessionHandler)
[2019-08-28 15:06:17,065] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Stopped (kafka.server.ReplicaFetcherThread)
[2019-08-28 15:06:17,066] INFO [ReplicaFetcher replicaId=0, leaderId=1, fetcherId=0] Shutdown completed (kafka.server.ReplicaFetcherThread)
[2019-08-28 15:06:17,074] INFO [ReplicaFetcher replicaId=0, leaderId=2, fetcherId=0] Shutdown completed (kafka.server.ReplicaFetcherThread)
[2019-08-28 15:06:17,078] INFO [ReplicaFetcherManager on broker 0] shutdown completed (kafka.server.ReplicaFetcherManager)
[2019-08-28 15:06:17,078] INFO [ReplicaAlterLogDirsManager on broker 0] shutting down (kafka.server.ReplicaAlterLogDirsManager)
[2019-08-28 15:06:17,079] INFO [ReplicaAlterLogDirsManager on broker 0] shutdown completed (kafka.server.ReplicaAlterLogDirsManager)
[2019-08-28 15:06:17,079] INFO [ExpirationReaper-0-Fetch]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:17,160] INFO [ExpirationReaper-0-Fetch]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:17,160] INFO [ExpirationReaper-0-Fetch]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:17,160] INFO [ExpirationReaper-0-Produce]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:17,336] INFO [ExpirationReaper-0-Produce]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:17,336] INFO [ExpirationReaper-0-Produce]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:17,336] INFO [ExpirationReaper-0-DeleteRecords]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:17,541] INFO [ExpirationReaper-0-DeleteRecords]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:17,542] INFO [ExpirationReaper-0-DeleteRecords]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:17,542] INFO [ExpirationReaper-0-ElectPreferredLeader]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:17,567] INFO [ExpirationReaper-0-ElectPreferredLeader]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:17,567] INFO [ExpirationReaper-0-ElectPreferredLeader]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2019-08-28 15:06:17,582] INFO [ReplicaManager broker=0] Shut down completely (kafka.server.ReplicaManager)
[2019-08-28 15:06:17,583] INFO Shutting down. (kafka.log.LogManager)
[2019-08-28 15:06:24,239] WARN [Controller id=0, targetBrokerId=0] Connection to node 0 (/192.168.1.13:9092) could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2019-08-28 15:06:25,531] INFO Shutdown complete. (kafka.log.LogManager)
[2019-08-28 15:06:25,557] INFO [ZooKeeperClient Kafka server] Closing. (kafka.zookeeper.ZooKeeperClient)
[2019-08-28 15:06:26,096] INFO EventThread shut down for session: 0x1000e9ed7a60004 (org.apache.zookeeper.ClientCnxn)
[2019-08-28 15:06:26,095] INFO Session: 0x1000e9ed7a60004 closed (org.apache.zookeeper.ZooKeeper)
[2019-08-28 15:06:26,102] INFO [ZooKeeperClient Kafka server] Closed. (kafka.zookeeper.ZooKeeperClient)
[2019-08-28 15:06:26,103] INFO [ThrottledChannelReaper-Fetch]: Shutting down (kafka.server.ClientQuotaManager$ThrottledChannelReaper)
[2019-08-28 15:06:26,731] INFO [ThrottledChannelReaper-Fetch]: Stopped (kafka.server.ClientQuotaManager$ThrottledChannelReaper)
[2019-08-28 15:06:26,731] INFO [ThrottledChannelReaper-Fetch]: Shutdown completed (kafka.server.ClientQuotaManager$ThrottledChannelReaper)
[2019-08-28 15:06:26,732] INFO [ThrottledChannelReaper-Produce]: Shutting down (kafka.server.ClientQuotaManager$ThrottledChannelReaper)
[2019-08-28 15:06:27,670] INFO [ThrottledChannelReaper-Produce]: Stopped (kafka.server.ClientQuotaManager$ThrottledChannelReaper)
[2019-08-28 15:06:27,670] INFO [ThrottledChannelReaper-Produce]: Shutdown completed (kafka.server.ClientQuotaManager$ThrottledChannelReaper)
[2019-08-28 15:06:27,682] INFO [ThrottledChannelReaper-Request]: Shutting down (kafka.server.ClientQuotaManager$ThrottledChannelReaper)
[2019-08-28 15:06:27,732] INFO [ThrottledChannelReaper-Request]: Shutdown completed (kafka.server.ClientQuotaManager$ThrottledChannelReaper)
[2019-08-28 15:06:27,732] INFO [ThrottledChannelReaper-Request]: Stopped (kafka.server.ClientQuotaManager$ThrottledChannelReaper)
[2019-08-28 15:06:27,796] INFO [SocketServer brokerId=0] Shutting down socket server (kafka.network.SocketServer)
[2019-08-28 15:06:27,931] INFO [SocketServer brokerId=0] Shutdown completed (kafka.network.SocketServer)
[2019-08-28 15:06:28,014] INFO [KafkaServer id=0] shut down completed (kafka.server.KafkaServer)
